modules/data_release/ajax/AddPermission.php

```diff
@@ -20,9 +20,27 @@
 $baseURL  = $settings->getBaseURL();
 $DB       = $factory->database();
 
-if (isset($_POST['action']) && $_POST['action'] == 'addpermission'
-    && $user->hasPermission('data_release_edit_file_access')
-) {
+
+if (!isset($_GET['action'])) {
+    header('');
```

Comment:
```suggestion
    http_response_code(400);
```

This should probably return a Bad Request if there is no action.